modules/images/webp-uploads/readme.txt

```diff
@@ -0,0 +1,63 @@
+=== Performance Lab - WebP Uploads Module ===
+
+Contributors:      wordpressdotorg
+Requires at least: 6.1
+Tested up to:      6.1
+Requires PHP:      5.6
+Stable tag:        1.0.0
+License:           GPLv2 or later
+License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+Tags:              performance, images, site health, measurement, webp
+
+WebP uploads module for the [performance plugin](https://wordpress.org/plugins/performance-lab/) from the WordPress Performance Team.
```

Comment:
I don't think we should mention the Performance Lab plugin here given that this is the description for the standalone plugin. I think we can use the description that is also present in the module header:
```suggestion
Creates WebP versions for new JPEG image uploads if supported by the server.
```